server/src/rbx_snapshot.rs

```diff
@@ -351,6 +372,20 @@ fn snapshot_imfs_directory<'source>(
     Ok(Some(snapshot))
 }
 
+#[derive(Debug, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct InitMetaJson {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    class_name: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    ignore_unknown_children: Option<bool>,
+    #[serde(
+        default = "HashMap::new",
+        skip_serializing_if = "HashMap::is_empty",
+    )]
+    properties: HashMap<String, RbxValue>,
```

Comment:
Do you want to add an empty line before each property that has an attribute?

I've found that it's easier to read structs when they're spaced out once attributes like these are involved.